Documentation/api_v3.md

```diff
@@ -2,6 +2,30 @@
 
 TODO: finish API doc
 
+## Response Header
+
+All Responses from etcd API has a [response header][response_header] attached. Response header includes the metadata of the response.
+
+```proto
+message ResponseHeader {
+  uint64 cluster_id = 1;
+  uint64 member_id = 2;
+  int64 revision = 3;
+  uint64 raft_term = 4;
+}
+```
+
+* Cluster_ID - the ID of the cluster that generates the response
+* Member_ID - the ID of the member that generates the response
+* Revision - the revision of the key-value store when the response is generated
+* Raft_Term - the Raft term of the member when the response is generated
+
+Applications can utilize `Cluster_ID` (`Member_ID`) field to ensure always talking to the same cluster (member). 
```

Comment:
An application may utilize the `Cluster_ID` (`Member_ID`) field to ensure it is communicating with the intended cluster (member).